### ISSUE TITLE
New PYPI release request

### ISSUE THREAD

**Issue by JohnLunzer** [grantjenks/python-sortedcontainers#241]
Please release a new version on PYPI. Thank you.

**Comment by jlunzer** [comment]
Bump

**Comment by jnrbsn** [comment]
@grantjenks Are you still maintaining this project? It's been almost four years since a new release was published on PyPI, and 44 commits to the repo have been made since then. Also, it doesn't look like you've responded to any issues or pull requests in over a year. Can you at least merge #247, which adds Python 3.13 to the build, and then create a new release on PyPI?

**Comment by theandrelima** [comment]
I think it might be time for someone to step up as the maintainer of a new project forked from this one.

**Comment by jnrbsn** [comment]
@grantjenks Can you at least just leave any sort of comment indicating whether you're still willing to maintain this project? It's unclear if you've even seen our comments.

**Comment by thebaptiste** [comment]
Yes, a new release integrating the addition of `pyproject.toml` would be appreciated.
`setup.py bdist-wheel` is legacy, deprecated, and slated for removal in pip 25.3 this year, see [here](https://github.com/pypa/pip/issues/6334)
